src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -88,6 +88,10 @@ public class BuildApk : AndroidTask
 		//[Required]
 		public bool EnableCompression { get; set; }
 
+		public bool IncludeWrapSh { get; set; }
+
+		public string CheckedBuild { get; set; }
```

Comment:
Should this be `bool`, or can it have more than two values?

Reply:
Nevermind, I guess this can be `asan` or `ubsan`.

Reply:
Yes, possibly other sanitizer names in the future